### PR TITLE
feat: simple-navbar

### LINKED ISSUE #41: feat: setup simple navbar
https://tailwindcss-radix.vercel.app/

### PR DESCRIPTION
**Pull request by ssarkar551**

### CODE REVIEW

**Comment by ssarkar551** [comment]
added simple navbar
image

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu";
+import clsx from "clsx";
+import React from "react";
+
+// type Props = {};
```
Comment (AyanavaKarmakar) [inline src/components/Navbar.tsx]:
```suggestion
```

File: src/components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu";
```
Comment (AyanavaKarmakar) [inline src/components/Navbar.tsx]:
```suggestion
import * as NavigationMenu from "@radix-ui/react-navigation-menu";
```

Comment (ssarkar551) [reply src/components/Navbar.tsx]: done

File: src/components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import * as NavigationMenu from "@radix-ui/react-navigation-menu";
+import clsx from "clsx";
+import React from "react";
```
Comment (AyanavaKarmakar) [inline src/components/Navbar.tsx]:
```suggestion
```

Not required in React v18

File: src/components/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+
```
Comment (AyanavaKarmakar) [inline src/components/Navbar.tsx:3]:
```suggestion
```

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

File: src/components/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            )}
+          >
+            <NavigationMenu.Trigger
+              className={clsx(
+                "rounded-md px-3 py-2 text-sm hover:bg-gray-100 dark:hover:bg-gray-900",
+                "text-sm font-medium text-gray-700 dark:text-gray-100",
+                "focus:outline-none focus-visible:ring focus-visible:ring-purple-500 focus-visible:ring-opacity-75",
+              )}
+            >
+              Resources
```
Comment (AyanavaKarmakar) [inline src/components/Navbar.tsx]:
```suggestion
              Profile
```

**Review by AyanavaKarmakar — APPROVED** [review]
LGTM

**Pull request merged**